<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>INFORMACIÓN DEL MODELO:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Código:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tecnologia:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDiseno">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSeguridad">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>90</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Seguridad:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnguardar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>GUARDAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnlimpiar">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LIMPIAR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCodigo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTecnologia">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Diseño:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Interior:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtInterior">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Precio:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPrecio">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ELIMINAR</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MODELO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>641</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>LISTADO DE MODELOS:</string>
    </property>
    <widget class="QTableWidget" name="tblModelos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>611</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Cod.Modelo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Diseño</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tecnologia</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Seguridad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Interior</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Precio</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
